.eslintrc.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     'promise/no-nesting': 0,
     'node/no-unsupported-features/es-syntax': 0,
     camelcase: 0,
+    'unicorn/prefer-prototype-methods': 0,
```

Comment:
We can remove this once https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1343 is fixed